source/Calamari.Tests/KubernetesFixtures/Terraform/Clusters/AKS-local-access-disabled/providers.tf

```diff
@@ -2,7 +2,7 @@ terraform {
   required_providers {
     azurerm = {
       source  = "hashicorp/azurerm"
-      version = "3.99.0" 
+      version = "3.95.0" 
```

Comment:
Can you also replace this in the other aks providers.tf. Just so they are the same